android/app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:usesCleartextTraffic="true"
```

Comment:
do we want this to stay in?

Reply:
Yes, this is a temporary workaround for the server giving http links